torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -74,7 +74,7 @@
     five_crop,
     five_crop_image_pil,
     five_crop_image_tensor,
-    hflip,
+    hflip,  # TODO: Consider moving all pure alias definitions at the bottom of the file
```

Comment:
I know we have discussed this previously and decided to keep aliases defined in their files (aka geometry, color etc). One issue with this is that we are mixing "real kernels" with aliases. Of course once the migration completes, differentiating between the two will be easy. An alias will point to a method that its `__name__` won't match. But now we are not able to tell the difference between the two. So because everything is defined on the same file, which version of the mid-level kernel will be kept depends on the alphanumeric order. So `hflip` is the name selected instead of `horizontal_flip` and `vertical_flip` instead of `vflip`.

This could be resolved by moving the alias definition here at the end of this file and have them all in one place. No super strong opinion about this and we can leave the discussion for later. I just added a TODO, so that we can review again this decision in case we want to do something different.